src/components/Card/CardFooter.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import { themr, ThemedComponentClass } from '@friendsofreactjs/react-css-themr';
+
+import { Action } from '../../types';
+import { buttonFrom } from '../Button';
+import ButtonGroup from '../ButtonGroup';
+import { CARD } from '../ThemeIdentifiers';
+import * as baseTheme from './Card.scss';
+import { classNames } from '@shopify/react-utilities/styles';
+
+export interface Props {
+  // Card footer related components to render inside this card footer.
+  children?: React.ReactNode;
+  // Primary action in the card footer 
+  primaryFooterAction?: Action;
+  // Secondary action in the card footer
+  secondaryFooterAction?: Action;
+  // Custom style
+  componentStyle?: any;
+  // Custom class
+  componentClass?: string;
+  // Theme to be injected via css-themr.
+  theme?: any;
+}
+
+const cardFooter = ({ children, primaryFooterAction, secondaryFooterAction, theme, componentStyle, componentClass  }: Props) => {
+
+  const primaryFooterActionMarkup = primaryFooterAction
+    ? buttonFrom(primaryFooterAction, { primary: true })
+    : null;
+
+  const secondaryFooterActionMarkup = secondaryFooterAction
+    ? buttonFrom(secondaryFooterAction)
+    : null;
+
+  const footerMarkup = primaryFooterActionMarkup || secondaryFooterActionMarkup
+    ? (
+      <div className={theme.footer}>
+        <ButtonGroup>
```

Comment:
@HiirenP its not necessary that footer will have buttons, user can put anything there, keep it generic so it can accept any type of element or component, its just should be separate from header & body

Reply:
Yes you are correct but this is for predefined buttons(like ok, cancel / save, cancel etc..) and this will be rendered only if we give them a primary or secondary action via props, otherwise it any react component as child will be rendered. which is in else condition. and this was coded like this for card component before. should i still remove it?

Reply:
@dheeraja00  Please see comments above i have added

Reply:
> Yes you are correct but this is for predefined buttons(like ok, cancel / save, cancel etc..) and this will be rendered only if we give them a primary or secondary action via props, otherwise it any react component as child will be rendered. which is in else condition. and this was coded like this for card component before. should i still remove it?

Hmm, yeah I think let's remove it because this will create more complications, like what should happen on click of those buttons, if a user wants to add an extra style to those buttons or call specific functions on click or disable it. So its better to remove it